### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by CindyDorantes**
## Project Requirements 🎯  🎯 
- Add the following validations:
  - For the User model:
    - Name must not be blank.
    - PostsCounter must be an integer greater than or equal to zero.
  - For the Post model:
    - Title must not be blank.
    - Title must not exceed 250 characters.
    - CommentsCounter must be an integer greater than or equal to zero.
    - LikesCounter must be an integer greater than or equal to zero.
- Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: spec/models/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    it 'title must not be blank' do
+      subject.title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'Title must not exceed 250 characters' do
+      long_title = 'long ' * 55
+      subject.title = long_title
+      expect(subject).to_not be_valid
+    end
```
Comment (Davidosky007) [inline spec/models/post_spec.rb:29]: * Kindly Add unit specs files for all of your models' methods and validations. according to the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)  ( ie Has unit tests for `User`, `Post`, `Comment`, and `Like` model )

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @CindyDorantes 👋, 
you-can-do-this-gif

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Good looking reamde file 👍 
* Well written PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/post_spec.rb (hunk 1 above)
Comment (CindyDorantes) [reply spec/models/post_spec.rb:29]: Sure! An apologize, I completely forgot about it 😞

**Review by rustam-tolipov — APPROVED** [review]
Hi @CindyDorantes, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**